packages/drivers/odsp-driver/src/resolveDataStore.ts

```diff
@@ -0,0 +1,19 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { getLocatorFromOdspUrl } from "./odspFluidFileLink";
+
+/**
+ * Retrieves data store path information from a storage URL. Returns undefined if the resolver does not handle this URL
+ */
+export function resolveDataStore(url: URL): string | undefined {
```

Comment:
Same thing about this needing to be accessible to components :)

Reply:
Will this be blocking for sharing link integration before host can consume this resolver?

Reply:
Nope, we can keep the same hacks/wrappers we have today but we won't be able to migrate completely off of our logic until this is accessible to components